src/routes/HomePage.jsx

```diff
@@ -2,14 +2,16 @@ import CategoryGrid from "../components/Grids/CategoryGrid";
 
 const HomePage = () => {
   return (
-    <div className='category'>
-      <div>
-        <h1 className='h3 category-title'>Watch music videos</h1>
-        <div className='category-grid'>
-          <CategoryGrid />
+    <>
```

Comment:
> ![image](https://user-images.githubusercontent.com/67889207/163975872-f6fda9f9-0ad5-4f8d-9f01-90af61ec5156.png)
>  ![image](https://user-images.githubusercontent.com/67889207/163975913-cfc9f3e0-5307-4f22-adbe-a1889a198465.png)
Because i have a large screen  the image is taking less space and card is growing bigger because of which i have extra space on the right side of the image maybe give max-width.